sigle/src/pages-lib/_error.tsx

```diff
@@ -1,38 +1,136 @@
-import React from 'react';
+import React /*{ useEffect }*/ from 'react';
+import { useEffect } from 'react';
 import { NextPageContext } from 'next';
 import NextErrorComponent from 'next/error';
-import styled from 'styled-components';
-import tw from 'twin.macro';
+import { styled } from '../stitches.config';
 import Link from 'next/link';
 import * as Sentry from '@sentry/nextjs';
-import { Button, Container } from '../components';
-import { sigleConfig } from '../config';
+import { Button, Text } from '../ui';
+import { Container } from '../ui';
+import Image from 'next/image';
+import { useAuth } from '../modules/auth/AuthContext';
 
-const NotFoundContainer = styled(Container)`
-  ${tw`mt-8 mb-8 flex flex-col items-center text-center md:flex-row md:text-left`};
-`;
+const NotFoundContainer = styled(Container, {
```

Comment:
Right now the content is just below the header, what do you think about having it centered in the bottom block? Similar to how the Login page is behaving (without the header for the login but same idea). Should be possible to do it easily using flex.

![Capture d’écran 2022-01-18 à 13 12 42](https://user-images.githubusercontent.com/5749437/149935982-4ed569fc-6bfa-4c8f-b1c6-01eae890ada1.png)